task08/task08/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task08
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+           Console.WriteLine("Введите значение х");
+           double x = double.Parse(Console.ReadLine());
+           Console.WriteLine(Calc(x));
+        }
+        static double Calc(double x)
```

Comment:
Разделяйте методы в классе пустой строкой. Так код читается легче.
```suggestion
        }
        
        static double Calc(double x)
```